components/eamxx/tests/uncoupled/cosp/CMakeLists.txt

```diff
@@ -20,3 +24,33 @@ configure_file (${CMAKE_CURRENT_SOURCE_DIR}/input.yaml
                 ${CMAKE_CURRENT_BINARY_DIR}/input.yaml)
 configure_file (${CMAKE_CURRENT_SOURCE_DIR}/output.yaml
                 ${CMAKE_CURRENT_BINARY_DIR}/output.yaml)
+
+if (SCREAM_ENABLE_BASELINE_TESTS)
+  # Compare one of the output files with the baselines.
+  # Note: for other tests we do np1-vs-npX bfb tests, which is why one is enough.
+  #       COSP, is not bfb w.r.t. num ranks though. Still, we only check 1 output files against baselines
```

Comment:
@brhillman side question for you. I had originally added np1-vs-npX tests for cosp_standalone, but they failed, and I remembered we found this with PEM/ERP tests before. What did we decide? Are we planning to make COSP pem-invariant?